files/zh-cn/web/javascript/reference/global_objects/map/clear/index.md

```diff
@@ -21,20 +21,20 @@ myMap.clear();
 
 ## 示例
 
-### 调用`clear`方法
+### 调用 `clear()` 方法
 
 ```js
-var myMap = new Map();
-myMap.set("bar", "baz");
-myMap.set(1, "foo");
+const myMap = new Map();
+myMap.set('bar', 'baz');
+myMap.set(1, 'foo');
 
-myMap.size;       // 2
-myMap.has("bar"); // true
+console.log(myMap.size);       // 2
+console.log(myMap.has('bar')); // true
```

Comment:
please follow english document, if you want to improve it, please modify the english document first:

```suggestion
myMap.size;       // 2
myMap.has('bar'); // true
```